ODataLib/EdmLib/Desktop/.Net4.0/Microsoft/Data/Edm/Library/EdmStructuredType.cs

```diff
@@ -30,11 +30,6 @@ namespace Microsoft.Data.Edm.Library
     /// </summary>
     public abstract class EdmStructuredType : EdmType, IEdmStructuredType
     {
-        /// <summary>

```

Comment:
@hfaran This branch only contains parts of the source code. The whole project is located at ADO: https://identitydivision.visualstudio.com/OData/_git/maintenance-OData.5.x?path=%2Fndp%2Ffx%2Fsrc%2FDataEdmLib%2FMicrosoft%2FData%2FEdm%2FLibrary%2FEdmStructuredType.cs

Can you access this? If not, please let me know.

we'd love to see the PR and move it forward.